### PR TITLE
Added notice for Neve Pro in the page builder module

### PR DESCRIPTION
**Pull request by arinaturcu**
If Neve is active and the user is not pro, a notice is added in the Page Builder module's description.

### Test instructions
- The notice should appear in the description of the page builder module that is displayed when Elementor is active
- It should appear only if Neve is active and Neve Pro is not active

[Codeinwp/neve-pro-addon#1741](https://github.com/Codeinwp/neve-pro-addon/issues/1741)

### CODE REVIEW

**Review by cristian-ungureanu — CHANGES_REQUESTED** [review]

File: obfx_modules/elementor-widgets/init.php [hunk 1]
```diff
@@ -29,8 +29,21 @@ class Elementor_Widgets_OBFX_Module extends Orbit_Fox_Module_Abstract {
...
-		$this->name           = __( 'Page builder widgets', 'themeisle-companion' );
-		$this->description    = __( 'Adds widgets to the most popular builders: Elementor or Beaver. More to come!', 'themeisle-companion' );
+		$this->name        = __( 'Page builder widgets', 'themeisle-companion' );
+		$this->description = __( 'Adds widgets to the most popular builders: Elementor or Beaver. More to come!', 'themeisle-companion' );
+
+		if ( wp_get_theme()->get( 'Name' ) === 'Neve' && ! is_plugin_active( 'neve-pro-addon/neve-pro-addon.php' ) ) {
+			$this->description .=
+				'<div class="neve-pro-notice">
+					<p>' /* translators: %s are bold html tags */
+						. sprintf( __( 'You can get access to %1$s10+ more Elementor and Gutenberg widgets%2$s, including Instagram integration, display conditions and more in %3$sNeve PRO%4$s.', 'themeisle-companion' ), '<b>', '</b>', '<b>', '</b>' ) . '
```
Comment (cristian-ungureanu) [inline obfx_modules/elementor-widgets/init.php:39]: @arinaturcu, I think it's better to combine the strings like this so when translation happens, they won't have parameters in the ending results that are replaced just by HTML tags. Also, the `Neve PRO` string doesn't need a translation.

```
sprintf(
		__( 'You can get access to %1$s, including Instagram integration, display conditions and more in %2$s.', 'themeisle-companion' ).
		sprintf( '<b>%1$s</b>',
			__( '10+ more Elementor and Gutenberg widgets', 'themeisle-companion' ),
		),
		'<b>Neve PRO</b>'
		);
```

**Review by cristian-ungureanu — CHANGES_REQUESTED** [review]
Thank you for the changes @arinaturcu, just a little change is required and then we can move this to the testing column.

**Review by cristian-ungureanu — APPROVED** [review]

**Comment by cristian-ungureanu** [comment]
Looks good! Thank you @arinaturcu 🚀

**Comment by irinelenache** [comment]
@arinaturcu Tested this PR and didn't find any problem 🚀 Will move it to **Ready to merge**

**Pull request merged**